certified-connectors/Mural/apiDefinition.swagger.json

```diff
@@ -1,1074 +1,1394 @@
+										},
+										"contentEditedOn": {
+											"type": "integer",
+											"format": "int32",
+											"description": "content edited on"
+										},
+										"createdBy": {
+											"type": "object",
+											"properties": {
+												"firstName": {
+													"type": "string",
+													"description": "first name"
+												},
+												"lastName": {
+													"type": "string",
+													"description": "last name"
+												}
+											},
+											"description": "created by"
+										},
+										"createdOn": {
+											"type": "integer",
+											"format": "int32",
+											"description": "created on"
+										},
+										"height": {
+											"type": "number",
+											"format": "float",
+											"description": "height",
+											"title": ""
+										},
+										"hidden": {
+											"type": "boolean",
+											"description": "hidden"
+										},
+										"hideEditor": {
+											"type": "boolean",
+											"description": "hide editor"
+										},
+										"hideOwner": {
+											"type": "boolean",
+											"description": "hide owner"
+										},
+										"id": {
+											"type": "string",
+											"description": "id"
+										},
+										"instruction": {
+											"type": "string",
+											"description": "instruction"
+										},
+										"locked": {
+											"type": "boolean",
+											"description": "locked"
+										},
+										"lockedByFacilitator": {
+											"type": "boolean",
+											"description": "locked by facilitator"
+										},
+										"parentId": {
+											"type": "string",
+											"description": "parent id"
+										},
+										"presentationIndex": {
+											"type": "integer",
+											"format": "int32",
+											"description": "presentation index"
+										},
+										"rotation": {
+											"type": "integer",
+											"format": "int32",
+											"description": "rotation"
+										},
+										"stackingOrder": {
+											"type": "integer",
+											"format": "int32",
+											"description": "stacking order"
+										},
+										"title": {
+											"type": "string",
+											"description": "title"
+										},
+										"updatedBy": {
+											"type": "object",
+											"properties": {
+												"firstName": {
+													"type": "string",
+													"description": "first name"
+												},
+												"lastName": {
+													"type": "string",
+													"description": "last name"
+												}
+											},
+											"description": "updated by"
+										},
+										"updatedOn": {
+											"type": "integer",
+											"format": "int32",
+											"description": "updated on"
+										},
+										"width": {
+											"type": "integer",
+											"format": "int32",
+											"description": "width"
+										},
+										"x": {
+											"type": "number",
```

Comment:
@rmiller-mural The type has been changed for the output field. Existing flows may be broken if they are using this field and actual payload has been changed.
can you please confirm that no users will be affected by this change?